packages/core/src/Cardano/util/credentialsFromAddress.ts

```diff
@@ -0,0 +1,40 @@
+import { Hash28ByteBase16 } from '@cardano-sdk/crypto';
+import { Cardano } from '../../../dist/esm';
```

Comment:
This is going to cause trouble. Best to have multiple imports from `src/`, e.g. `from '../Address'`.